src/wwwroot/app/views/vacancy/vacancy.edit.controller.js

```diff
@@ -172,16 +173,14 @@ export default function VacancyController( //eslint-disable-line max-statements
    function _vs() {
       let memo = vm.vacancy.comments;
       vm.vacancy.comments = vm.comments;
-      VacancyService.save(vm.vacancy).then(vacancy => {
-         vm.vacancy = vacancy;
-         vm.comments = cloneDeep(vm.vacancy.comments);
-         UserDialogService.notification($translate.instant('DIALOG_SERVICE.SUCCESSFUL_SAVING'), 'success');
-         SearchService.invalidateVacancies();
-      })
-         .then(() => $state.go(
-               $state.previous.name || DEFAULT_REDIRECT_VIEW_NAME,
-               {_data: vm.vacancy, vacancyId: vm.vacancy.id},
-               { reload: true }))
+      VacancyService.save(vm.vacancy)
+         .then(vacancy => {
+            vm.vacancy = vacancy;
+            vm.comments = cloneDeep(vm.vacancy.comments);
+            UserDialogService.notification($translate.instant('DIALOG_SERVICE.SUCCESSFUL_SAVING'), 'success');
+            SearchService.invalidateVacancies();
+            TransitionsService.back(DEFAULT_REDIRECT_VIEW_NAME, {vacancyId : vacancy.id});
```

Comment:
think if this nice change, because in the previous case it was going in separate then